assets/tests/components/map.test.tsx

```diff
@@ -234,6 +234,64 @@ describe("map", () => {
       screen.queryByRole("link", { name: /Go to Street View/ })
     ).not.toBeInTheDocument()
   })
+
+  test("can turn on street view and click on the map", async () => {
+    const openSpy = jest.spyOn(window, "open").mockImplementationOnce(jest.fn())
```

Comment:
suggestion/question: I'm having trouble coming up with a name, but I think this should describe that street view should disable itself when a point is clicked.

Reply:
Yeah I wanted to write a better name but was struggling to come up with something that isn't an absurd run-on sentence.

Reply:
I'd also be cool with just adding a comment near the end of the test that explains that constraint. 

> struggling to come up with something that isn't an absurd run-on sentence.

Me too! 😭

Reply:
> struggling to come up with something that isn't an absurd run-on sentence.

It's like the test framework is trying to get us to write two separate (basically identical) tests lol